# Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
# SPDX-License-Identifier: BSD-3-Clause-Clear

float32 usage
uint64 user
uint64 nice
uint64 system
uint64 idle
uint64 iowait
uint64 irq
uint64 softirq
uint64 steal
uint64 guest
uint64 guest_nice